tests/cross_functional/system_test/test_mcg_bucket_notification.py

```diff
@@ -0,0 +1,300 @@
+            skip_any_features=["nsfs", "rgw kafka", "caching"],
+        )
+
+        # 1. Enable bucket notification on Noobaa CR
+        notify_manager.enable_bucket_notifs_on_cr(use_provided_pvc=True)
+
+        # 2. Create kafka topic and add it to the Noobaa CR
+        topic = notify_manager.create_kafka_topic()
+        secret, conn_config_path = notify_manager.create_kafka_conn_secret(topic)
+        notify_manager.add_notif_conn_to_noobaa_cr(secret)
+
+        # 3. Create the five buckets that is needed for the testing
+        buckets_created = []
+        for i in range(NUM_OF_BUCKETS):
+            bucket = bucket_factory()[0]
+            logger.info(f"Enabling bucket notification for the bucket {bucket.name}")
+            notify_manager.put_bucket_notification(
+                awscli_pod,
+                mcg_obj,
+                bucket.name,
+                events=events_to_notify,
+                conn_config_path=conn_config_path,
+            )
+            buckets_created.append(bucket.name)
+
+        # 4. Write some object to all the buckets and verify the events
+        # has occurred for all the buckets
+        obj_written = []
+        for bucket in buckets_created:
+            obj_written = write_random_test_objects_to_bucket(
+                awscli_pod,
+                bucket,
+                file_dir=test_directory_setup.origin_dir,
+                amount=2,
+                mcg_obj=mcg_obj,
+            )
+        self.verify_events(
+            notify_manager,
+            topic,
+            bucket_names=buckets_created[:],
+            event_name="ObjectCreated:Put",
+        )
+
+        with ThreadPoolExecutor(max_workers=5) as executor:
+
+            # 5. Tag object from the bucket and restart noobaa pod nodes
+            # Verify ObjectTagging:Put events has occurred for all the buckets
+            noobaa_pods = [
+                get_noobaa_core_pod(),
+                get_noobaa_db_pod(),
+            ] + get_noobaa_endpoint_pods()
+            noobaa_pod_nodes = [get_pod_node(pod_obj) for pod_obj in noobaa_pods]
+            aws_cli_pod_node = get_pod_node(awscli_pod).name
+            for node_obj in noobaa_pod_nodes:
+                if node_obj.name != aws_cli_pod_node:
+                    node_to_shutdown = [node_obj]
+                    break
+
+            future_objs = []
+            for bucket in buckets_created:
+                logger.info(f"Tagging object {obj_written[1]} from the bucket {bucket}")
+                future_objs.append(
+                    executor.submit(
+                        s3_put_object_tagging,
+                        mcg_obj,
+                        bucket,
+                        obj_written[1],
+                        [{"Key": "type", "Value": "media"}],
+                    )
+                )
+
+            logger.info(f"Stopping Noobaa pod node {node_to_shutdown}")
+            nodes.stop_nodes(nodes=node_to_shutdown)
```

Comment:
stop_nodes by default forces the instance to stop. It will ungraceful stop the instance which is not equivalent to a graceful shutdown

Reply:
It doesnt matter in this case. we are not looking for graceful shutdown also.